pkg/driverbuilder/builder/amazonlinux.go

```diff
@@ -327,21 +327,16 @@ func bunzip(data io.Reader) (res []byte, err error) {
 }
 
 func amazonGccVersionFromKernelRelease(kr kernelrelease.KernelRelease) string {
-	switch kr.Version {
```

Comment:
Interesting that this works, I thought that `gcc-10` was required to build the drivers.

Reply:
Can you point to where it states this is required? They all seemed to build fine with gcc-8 and gcc-10 didn't seem to be pre-installed in the image when I checked.